modules/web/client/src/main/scala/observe/ui/components/sequence/SequenceTableDefs.scala

```diff
@@ -0,0 +1,259 @@
+    XBinColumnId            -> FixedSize(60.toPx),
+    YBinColumnId            -> FixedSize(60.toPx),
+    ROIColumnId             -> Resizable(75.toPx, min = 75.toPx.some, max = 130.toPx.some),
+    SNColumnId              -> Resizable(75.toPx, min = 75.toPx.some, max = 130.toPx.some),
+    CameraColumnId          -> Resizable(10.toPx),
+    DeckerColumnId          -> Resizable(10.toPx),
+    ReadModeColumnId        -> Resizable(180.toPx),
+    ImagingMirrorColumnId   -> Resizable(10.toPx),
+    DatasetsColumnId        -> Resizable(300.toPx, min = 75.toPx.some),
+    SettingsColumnId        -> FixedSize(39.toPx)
+  )
+
+  // The order in which they are removed by overflow. The ones at the beginning go first.
+  // Missing columns are not removed by overflow.
+  protected val ColumnPriorities: List[ColumnId] = List(
+    PColumnId,
+    QColumnId,
+    GuideColumnId,
+    ExposureColumnId,
+    SNColumnId,
+    ROIColumnId,
+    XBinColumnId,
+    YBinColumnId,
+    FilterColumnId,
+    GratingColumnId,
+    FPUColumnId,
+    CameraColumnId,
+    DeckerColumnId,
+    ReadModeColumnId,
+    ImagingMirrorColumnId,
+    SettingsColumnId,
+    DatasetsColumnId
+  ).reverse
+
+  val DynTableDef = DynTable(
+    ColumnSizes,
+    ColumnPriorities,
+    DynTable.ColState(
+      resized = ColumnSizing(),
+      visibility = ColumnVisibility(
+        CameraColumnId        -> Visibility.Hidden,
+        DeckerColumnId        -> Visibility.Hidden,
+        ReadModeColumnId      -> Visibility.Hidden,
+        ImagingMirrorColumnId -> Visibility.Hidden
+      )
+    )
+  )
+
+  private def flippedBreakpoint(breakpoint: Breakpoint): Breakpoint =
+    breakpoint match
+      case Breakpoint.Enabled  => Breakpoint.Disabled
+      case Breakpoint.Disabled => Breakpoint.Enabled
+
+  private def column[V](
+    id:     ColumnId,
+    header: VdomNode,
+    cell:   js.UndefOr[
+      raw.buildLibCoreCellMod.CellContext[SequenceTableRowType, V] => VdomNode
+    ] = js.undefined
+  ): ColumnDef[SequenceTableRowType, V] =
+    ColDef[V](id, header = _ => header, cell = cell).setColumnSize(ColumnSizes(id))
+
+  protected def columnDefs(flipBreakpoint: (Observation.Id, Step.Id, Breakpoint) => Callback)(
+    clientMode:     ClientMode,
+    instrument:     Instrument,
+    obsId:          Observation.Id,
+    requests:       ObservationRequests,
+    executionState: ExecutionState,
+    progress:       Option[StepProgress],
+    isPreview:      Boolean,
+    selectedStepId: Option[Step.Id]
+  ): List[ColumnDef[SequenceTableRowType, ?]] =
+    List(
+      SequenceColumns.headerCell(HeaderColumnId, ColDef).setColumnSize(ColumnSizes(HeaderColumnId)),
+      column(
+        BreakpointColumnId,
+        "",
+        cell =>
+          cell.row.original.value.toOption.map: stepRow =>
+            val step: SequenceRow[DynamicConfig] = stepRow.step
+            val stepId: Option[Step.Id]          = step.id.toOption
+            // val canSetBreakpoint =
```

Comment:
Commented code

Reply:
This code was already commented, it was just moved to another file. It's code from seqexec for which we still have to migrate the logic.